android/java/org/chromium/chrome/browser/BraveFeatureList.java

```diff
@@ -18,6 +18,8 @@ public abstract class BraveFeatureList {
     public static final String ENABLE_FORCE_DARK = "enable-force-dark";
     public static final String ENABLE_TAB_GROUPS = "enable-tab-groups";
     public static final String ENABLE_TAB_GRID = "enable-tab-grid-layout";
+    public static final String BRAVE_WALLET_SOLANA = "BraveWalletSolana";
+    public static final String Brave_Wallet_Solana_Provider = "BraveWalletSolanaProvider";
```

Comment:
what is that for?

Reply:
`BraveWalletSolana` is to check if Sonala is enabled (guess you know) and added `Brave_Wallet_Solana_Provider` flag, I thought I might need it later, do you want it to be removed? LMK

Reply:
yeah, I asked for `Brave_Wallet_Solana_Provider`. What do you plan it for? Anyways, let's remove or comment with an explanation why do we need it in the future.